tests/beaker_tests/README-beaker-script-ref.md

```diff
@@ -0,0 +1,232 @@
+# Beaker Test Script Guide to Test Inputs
```

Comment:
1) I was trying to keep the filename short - it was intended to mean 'reference'. 
2) I thought about putting it in doc/ but liked the handiness of having the yaml doc in cmd_ref/, so I kept this one in beaker_tests/.

In the long run this document needs to be part of README-develop-beaker-scripts.md anyway, so I'll get rid of this document and add this content as an addendum to the main doc.